modules/ethereumNode.js

```diff
@@ -527,7 +517,7 @@ class EthereumNode extends EventEmitter {
         try {
             return fs.readFileSync(fullPath, {encoding: 'utf8'});
         } catch (err){
-            log.error(`Unable to read from ${fullPath}`, err);
+            log.error(`Unable to read from ${fullPath}`);
```

Comment:
It would be good to see an error if it's due to file being unreadable, etc. What we can do is do a pre-check using `fs.accessSync(fullPath, fs.R_OK)` - if this throws then it means the file can't be read. If it passes then we go ahead and do `fs.readFileSync`. If that fails we output the error.